skafka/src/test/scala/com/evolutiongaming/skafka/consumer/RebalanceCallbackSpec.scala

```diff
@@ -0,0 +1,219 @@
+package com.evolutiongaming.skafka.consumer
+
+import java.lang.{Long => LongJ}
+import java.time.{Duration => DurationJ}
+import java.util.{Collection => CollectionJ, Map => MapJ, Set => SetJ}
+
+import cats.effect.IO
+import com.evolutiongaming.skafka.consumer.DataPoints._
+import com.evolutiongaming.skafka.consumer.RebalanceCallback._
+import com.evolutiongaming.skafka.consumer.RebalanceCallbackSpec._
+import com.evolutiongaming.skafka.{Offset, Partition, TopicPartition}
+import org.apache.kafka.clients.consumer.{Consumer => ConsumerJ}
+import org.apache.kafka.common.{TopicPartition => TopicPartitionJ}
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+
+import scala.util.control.NoStackTrace
+import scala.util.{Failure, Try}
+
+class RebalanceCallbackSpec extends AnyFreeSpec with Matchers {
+
+  "RebalanceCallback" - {
+
+    "consumer unrelated methods do nothing with consumer" - {
+      val consumer: ConsumerJ[_, _] =
+        null // null to verify zero interactions with consumer, otherwise there would be an NPE
```

Comment:
summing @rtar :)

Reply:
summon success!
 will get rid of `null` and `var`s